# Helper message for storing maps
string key
string value
